src/proxy-ui-api/src/main/java/org/niis/xroad/restapi/service/TokenCertificateService.java

```diff
@@ -401,7 +420,7 @@ public CertificateInfo importCertificate(byte[] certificateBytes) throws Invalid
      * @throws InvalidCertificateException
      * @throws AccessDeniedException
      */
-    public void checkCertificateAuthority(byte[] certificateBytes) throws InvalidCertificateException,
+    public void verifyAuthorityToCertificate(byte[] certificateBytes) throws InvalidCertificateException,
```

Comment:
Maybe name could still be changed a bit. `verifyActivateDisableAuthority(certificateBytes)`?
There is already another type of authority to certificate -verification happening in this service, IMPORT_AUTH_CERT and IMPORT_SIGN_CERT authority on lines 306 and 312. It would be good if the method name would make it clear that this method does not handle that type of "authority to certificate".